index.js

```diff
@@ -0,0 +1,229 @@
+// Resume Builder
+const fs = require('fs');
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+let resume = {
+    "basics": {
+        "name": "",
+        "label": "",
+        "picture": "",
+        "email": "",
+        "phone": "",
+        "website": "",
+        "summary": ""
+      },
+      "work": [],
+      "education": [],
+      "skills": [],
+};
+let fileName = 'resume.json';
+
+console.log('==============================\nWelcome to Resume Builder!\n==============================');
+
+// Check if fileName is given (if not, use default)
+if (process.argv[2]) {
+    fileName = process.argv[2];
+    showCommands();
+} else {
+    showCommands();
+}
```

Comment:
I was thinking if showCommands() will be called regardless of whether a filename is given, then maybe we can just call it once but keeping the filename check separate, something like this:

if (process.argv[2]) {
    fileName = process.argv[2];
}
showCommands();